src/Turtle/Prelude.hs

```diff
@@ -575,6 +588,44 @@ touch file = do
 #endif
         else output file empty )
 
+type Permissions = Filesystem.Permissions -> Filesystem.Permissions
+
+-- | Update a file or directory's permissions
```

Comment:
Add a couple of example usages in the documentation side by side with the equivalent Bash:

```haskell
{-| Update a file or directory's permissions

> chmod rwo        "foo.txt"  -- chmod u=rw foo.txt
> chmod executable "foo.txt"  -- chmod u+x foo.txt
> chmod nonwritable "foo.txt" -- chmod u-x foo.txt
-}
```